### ISSUE TITLE
Pre-sets icons missing

### ISSUE THREAD

**Issue by stefantrav** [ksya/ha-nefiteasy#341]
hello when adding the thermostat card to the dashboard with the pre-set switch enabled you wont see any icons this very confusing

![image](https://github.com/user-attachments/assets/1107bd19-e7b7-47ae-a59f-12df08299eea)

**Comment by altserg** [comment]
This is the question to nefiteasy maintainers:
I can change the presets to HASS default like below so the UI will behave better:
Clock = ECO or COMFORT ?
Manual = COMFORT or maybe NONE ?
+ Add new AWAY preset that will activate the holiday mode 

But I need blessing that my pull request will be accepted

**Comment by RobBie1221** [comment]
I don't think this is a good solution, it makes things more confusing in my opinion. Keeping things consistent with the Nefit Easy app is I think the way to go to keep it clear for users what they should do. This should really be improved on HASS side.
